### DISCUSSION TITLE
CSS for Dark Mode Toggle

### DISCUSSION THREAD

**Discussion by jolo-dev** [prabhuignoto/react-chrono#240] [Q&A]
Hey folks,

Is it possible to inject a CSS class into the `react-chrono`?
I am using TailwindCSS and they have [Dark mode option](https://tailwindcss.com/docs/dark-mode).

Thanks in advance.

**Comment by prabhuignoto** [comment]
Yes this is possible. Please refer to the  [classNames](https://github.com/prabhuignoto/react-chrono#using-custom-class-names) section on how to do this.  Most of the elements can be targeted.
